pyiron/vasp/base.py

```diff
@@ -1662,11 +1705,12 @@ def from_hdf(self, hdf):
             self.incar.from_hdf(hdf5_input)
             self.kpoints.from_hdf(hdf5_input)
             self.potcar.from_hdf(hdf5_input)
+
+            self._eddrmm = "ignore"
```

Comment:
Isn't the default already specified? Why do you need this?

Reply:
we need this for older job without the tag. The default for new jobs is 'not_converged'